docs/src/main/paradox/java/http/client-side/client-transport.md

```diff
@@ -1 +0,0 @@
-../../../scala/http/client-side/client-transport.md
```

Comment:
this seems like a mistake - this should be just a symlink to the java one and have switches between the langs